src/models/AssetType.ts

```diff
@@ -0,0 +1,7 @@
+export type AssetType = "nfts" | "native-tokens";
```

Comment:
Likewise, the different asset types should also be pulled from a config file instead of being static in code.